modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/DropwizardServerGenerator.scala

```diff
@@ -0,0 +1,338 @@
+
+            val clsConstructor = cls.addConstructor(PROTECTED)
+            clsConstructor.addParameter(new Parameter(util.EnumSet.of(FINAL), PrimitiveType.intType, new SimpleName("statusCode")))
+            clsConstructor.setBody(
+              new BlockStmt(
+                new NodeList(
+                  new ExpressionStmt(new AssignExpr(new FieldAccessExpr(new ThisExpr, "statusCode"), new NameExpr("statusCode"), AssignExpr.Operator.ASSIGN))
+                )
+              )
+            )
+
+            val getStatusCodeMethod = cls.addMethod("getStatusCode", PUBLIC)
+            getStatusCodeMethod.setType(PrimitiveType.intType)
+            getStatusCodeMethod.setBody(new BlockStmt(new NodeList(
+              new ReturnStmt(new FieldAccessExpr(new ThisExpr, "statusCode"))
+            )))
+
+            cls
+          }
+
+          val responseClasses = responses.value.map { response =>
+            val clsName: String = response.statusCodeName.asString
+            val cls = new ClassOrInterfaceDeclaration(util.EnumSet.of(PUBLIC, STATIC), false, clsName)
+            cls.setExtendedTypes(new NodeList(abstractResponseClassType))
+
+            val (fields, constructor, methods) = response.value.fold({
+              val constructor = new ConstructorDeclaration(util.EnumSet.of(PUBLIC), clsName)
+              constructor.setBody(new BlockStmt(new NodeList(
+                new ExpressionStmt(new MethodCallExpr("super", new IntegerLiteralExpr(response.statusCode)))
+              )))
+              (List.empty[FieldDeclaration], constructor, List.empty[MethodDeclaration])
+            })({ case (valueType, _) =>
+              val valueField = new FieldDeclaration(util.EnumSet.of(PRIVATE, FINAL), new VariableDeclarator(valueType, "value"))
+
+              val constructor = new ConstructorDeclaration(util.EnumSet.of(PUBLIC), clsName)
+              constructor.addParameter(new Parameter(util.EnumSet.of(FINAL), valueType, new SimpleName("value")))
+              constructor.setBody(new BlockStmt(new NodeList(
+                new ExpressionStmt(new MethodCallExpr("super", new IntegerLiteralExpr(response.statusCode)))
+              )))
+
+              val getValueMethod = new MethodDeclaration(util.EnumSet.of(PUBLIC), valueType, "getValue")
+              getValueMethod.setBody(new BlockStmt(new NodeList(
+                new ReturnStmt(new FieldAccessExpr(new ThisExpr, "value"))
+              )))
+
+              (valueField :: Nil, constructor, getValueMethod :: Nil)
+            })
+            (fields ++ Option(constructor) ++ methods).foreach(cls.addMember)
+
+            cls
+          }
+          responseClasses.foreach(abstractResponseClass.addMember)
+
+          abstractResponseClass :: Nil
+        }
+
+      case RenderClass(className, handlerName, classAnnotations, combinedRouteTerms, extraRouteParams, responseDefinitions, supportDefinitions) =>
+        for {
+          _ <- safeParseSimpleName(className)
```

Comment:
These are just guards? The formatting would be ugly, but

```scala
def doRender: Option[A] = { val cls ...
safeParseSimpleName(className) >> safeParseSimpleName(handlerName) >> doRender
```

Reply:
yeah, just guards.  formatting is kinda eh, but yeah, i think i like that better.